### ISSUE TITLE
Checkpoint plugin not processing options correctly?

### ISSUE THREAD

**Issue by steindev** [ComputationalRadiationPhysics/picongpu#3977]
As noted during the crusher hackathon, the error
```bash
Unhandled exception of type 'St13runtime_error' with message 'There is no default value defined for the option file and idx is out of range', terminating
```
is thrown when using the checkpoint plugin.

cc'ing @franzpoeschel 

Using picongpu-dev as of today and TBG config attached
[tbg_checkpoint.md](https://github.com/ComputationalRadiationPhysics/picongpu/files/8036197/tbg_checkpoint.md)

**Comment by franzpoeschel** [comment]
I can reproduce this locally, checkpointing is currently entirely broken, very likely since merging #3720

**Comment by steindev** [comment]
Fixed by #3978
